Components/input/input.css

```diff
@@ -0,0 +1,67 @@
+.inputs{
+      display:flex;
+      flex-direction: column;
+      gap: 1.25rem;
+      align-items: center;
+      justify-content: center;
+      height: max-content;
+      padding: 1rem 4rem;
+      border: 1px solid #004085;
+      border-radius: 1rem;
+  }
+  
+  .inputs .input-field{
+      width: 70%;
+      height: 2rem;
+      padding-left: 1rem;
+      border-radius: .3rem;
+  }
```

Comment:
In here, you could use pixels instead of rems because they are whole numbers, and when decimals are used it seems like we are experimenting with it. Also, we wouldn't want to change such values with respect to font size.
Suggestion: If you are not purposely using it, could change it to pixels.